Automattic/wp-calypso
# Signup: Add plan specific signup flows for 3Y plans

Related to #74456

## Proposed Changes

* This adds signup flows for the 3Y Starter, Explorer, Creator and Entrepreneur plans.

## Testing Instructions



* Test the new plan flows:
  * /start/personal-3y
  * /start/premium-3y
  * /start/business-3y
  * /start/ecommerce-3y
* The flows should skip the plans step
* Checkout should contain the correct plan, with the 3-year duration

## Pre-merge Checklist
